views/show_rhapsody.handlebars

```diff
@@ -1,76 +1,42 @@
-                    didn’t
-                    “believe” to become “believers”; we believed to receive something: the God-life! In John 6:47,
-                    Jesus
-                    said,
-                    “...He that believeth on me hath everlasting life.” We believed so that we may possess “everlasting
-                    life”—the God-life. We’re more than believers. To elucidate this further, let’s look at it this
-                    way: If
-                    you
-                    took an entrance examination into a certain institution, and you were successful, you’d become a
-                    student,
-                    right? Sure! Having already been admitted as a student, you wouldn’t call yourself a candidate or
-                    an
-                    applicant to the school anymore, because you’re now a student! You were an applicant, so you could
-                    become a
-                    student. Likewise, we believed to become possessors of the God-life; we didn’t believe for nothing,
-                    or
-                    to
-                    be merely referred to as “believers.” Our opening verse says, “...whosoever believeth in him should
-                    not
-                    perish, but have everlasting life.” Every “believer” in Christ got, and became, something: we got
-                    eternal
-                    life, and became God-men and God-women. The Apostle Peter puts it rightly: we’re partakers of the
-                    divine
-                    nature; associates of the God-kind (2 Peter 1:4). You’ve transcended from a believer. Believing was
-                    the
-                    verb to take you into possessing everything that the death, burial and resurrection of Jesus Christ
-                    wrought
-                    on your behalf. No wonder the Bible says all things are yours (1 Corinthians 3:21). Take possession
-                    of
-                    all
-                    that’s yours in Christ and enjoy your triumphant life in Him. He came, not just to reveal the
-                    Father to
-                    us,
-                    but to bring us into oneness with Him, and make us gods! Hallelujah!
+                    {{ rhapsody.body }}
                 </p>
                 <h6>PRAYER</h6>
                 <P style="text-align:center; font-size:1.6rem;">
-                    All things are mine, because I’m the seed of Abraham, an heir of God, and a joint-heir with Christ.
-                    I walk, conscious of my prosperity, health, and victory, for in Christ, I’ve obtained an
-                    inheritance, having been predestined according to the purpose of Him who works all things according
-                    to the counsel of His own will. And by Him I reign, and rule in life! Blessed be God!
+                    {{ rhapsody.confession }}
```

Comment:
double braces will not escape html tags, but triple will
are we going to be saving the data inside db with html tags? ( which may make sense tbh )

if we are not, then we should use 3 braces
read [here](https://handlebarsjs.com/#html-escaping) for more info